source/uwp/Renderer/lib/AdaptiveActionRendererRegistration.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "AdaptiveCards.XamlCardRenderer.h"
+#include "Util.h"
+
+namespace AdaptiveCards { namespace XamlCardRenderer
+{
+    class AdaptiveActionRendererRegistration :
```

Comment:
>AdaptiveActionRendererRegistration [](start = 10, length = 34)

This class looks exactly the same as the ElementRenderer Registration. Sadly we can't inherit from templated BaseRegistration class that would make this class as easy as creating a BaseRegistration<ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveActionElement>

Reply:
The parser registrations will look the same as well.

---
In reply to: [141201602](https://github.com/Microsoft/AdaptiveCards/pull/695#discussion_r141201602) [](ancestors = 141201602)

Reply:
We spoke about this offline.  We can investigate further whether we can do something to reduce the duplicated code during the polish phase.

---
In reply to: [141201650](https://github.com/Microsoft/AdaptiveCards/pull/695#discussion_r141201650) [](ancestors = 141201650,141201602)